packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketLayoutNavigation.tsx

```diff
@@ -36,19 +36,25 @@ const NavListItem = styled(NavigationItem)`
             : ''}
 `;
 
-const Divider = styled.div`
+const StyledDivider = styled.div<{ elevation: Elevation }>`
     margin: ${spacingsPx.xxs} 0; /* gap xxs + margin xxs = spacing xs around divider */
     width: 100%;
-    border-bottom: 1px solid ${({ theme }) => theme.borderOnElevation0};
+    border-bottom: 1px solid ${({ theme, elevation }) => theme[mapElevationToBorder[elevation]]};
 
     ${variables.SCREEN_QUERY.BELOW_LAPTOP} {
         margin: 0 ${spacingsPx.xxs}; /* gap xxs + margin xxs = spacing xs around divider */
         flex-direction: row;
         border-bottom: 0;
-        border-right: 1px solid ${({ theme }) => theme.borderOnElevation0};
+        border-right: 1px solid ${({ theme, elevation }) => theme[mapElevationToBorder[elevation]]};
     }
 `;
 
+const Divider = () => {
+    const { elevation } = useElevation();
```

Comment:
This is super local, I would simply hard-code, do not see any benefit to using elevation here :(